### ISSUE TITLE
Upgrade from 7.11.x to 7.12 the icons are messed up

### ISSUE THREAD

**Issue by mabullo** [salesagility/SuiteCRM#9432]
In some SuiteCRM installations after the upgrade from 7.11.x to 7.12 the icons are messed up.
The html code is correct but they are misinterpreted.
I had opened the post in the SuiteCRM Community as collaboration and development
[](https://community.suitecrm.com/t/wrong-icons-after-upgrade-7-12-1/82997)

#### Issue
After updating SuiteCRM to version 7.12.1 I have some icons both in the administration menu, and in other places, completely wrong.
I always make a Quick Repair and Rebuild.
I delete the cache often and the problem remains.

#### Expected Behavior
After the upgrade the icons should remain the same

#### Actual Behavior
For example if I compare the following tag for the star icon in two of my installations, I get two different representations.
The html code is the same:
`<span class="suitepicon suitepicon-favorite-star-outline"></span>`
In the correct installation the icon is a [star](https://community.suitecrm.com/uploads/default/original/2X/8/880b1971db69491fe6591b3e861a1acad2c2bbe9.png), in the wrong one instead of the star I see an image of a [newspaper](https://community.suitecrm.com/uploads/default/original/2X/8/8fcc9cf4f7e365ae5ee616b517d1d29dfed286b4.png) that does not correspond at all to the class used in the span tag.

#### Steps to Reproduce
Try to upgrade from SuiteCRM version 7.11.x to 7.12.
I wrote **try** because as I explained in the community forum in other version upgrades for other installations I didn't have this problem.

#### Context
As I often do I have upgraded the version.
I believe it is of medium/high priority.

#### Your Environment

* SuiteCRM Version used:
* Browsers -> Chrome version  97.0.4692.71 | Firefox Dev version 97.0b4 (
* Server version: 10.3.32-MariaDB-0ubuntu0.20.04.1  - PHP 7.4.26
* Ubuntu 20.04

**Comment by SuiteBot** [comment]
This issue has been mentioned on **SuiteCRM**. There might be relevant details there:

https://community.suitecrm.com/t/wrong-icons-after-upgrade-7-12-1/82997/8

**Comment by mabullo** [comment]
I tried to overwrite the /themes folder with the folder taken from the new release but the problem doesn't change.
Maybe it's a sprites problem? 
Looking at the[ image](https://community.suitecrm.com/uploads/default/optimized/2X/f/f4f0d6c91fcc2534a1951c5cfc4e00eef2aad271_2_1035x555.png) you can see that in addition to the wrong icons, they also have the wrong vertical position.

**Comment by mabullo** [comment]
I solved it, thanks to another member of the community who had the same problem, on another topic, i realized that I had tried over and over again to overwrite the themes/SuiteP folder but actually the problem was in custom/themes/SuiteP, where I was carrying around an old version of themes/SuiteP as well as my changes in /custom/themes/SuiteP/tpls.
I deleted the unnecessary files and everything was fine.
It looked like a bug but it turned out to be my mistake.
